pkg/kubewarden/types/kubewarden.ts

```diff
@@ -29,6 +29,10 @@ export const KUBEWARDEN = {
   POLICY_SERVER:            'policies.kubewarden.io.policyserver'
 };
 
+export const KUBEWARDEN_CONFIGMAPS = {
+  COLLECTOR: 'kubewarden-collector',
```

Comment:
Detection does not work because collector name is `kubewarden-collector-09375fe8`.

```bash
~ kubectl get cm -n cattle-kubewarden-system | grep coll
kubewarden-collector-09375fe8             1      89m
```

Reply:
Do the annotations always reflect this dynamic name? There must be something we can relate to from the installed controller app.

Reply:
Just guessing, but here you are checking exact CM name?: https://github.com/rancher/kubewarden-ui/pull/917/commits/4aeaec334d8ed4084531fddbd6f2cb7205119a0d#diff-563a4e984f953d5969274d6ee93f8fec4a07cc4ac739f9f411c2388b1772f45dR218

Reply:
I've reverted this change and implemented a new change found in the latest `@rancher/shell` package to retrieve the app values.